oneview/resource_logical_enclosure.go

```diff
@@ -183,14 +191,17 @@ func resourceLogicalEnclosure() *schema.Resource {
 			"status": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
 			},
 			"type": {
 				Type:     schema.TypeString,
 				Optional: true,
```

Comment:
This is a read only field please remove optional from here.

Reply:
done